egs/wsj/s5/steps/libs/nnet3/xconfig/utils.py

```diff
@@ -85,6 +91,10 @@ def get_string_from_layer_name(all_layers, current_layer, full_layer_name):
     for layer in all_layers:
```

Comment:
Note that this comment supersedes all previous and following comments and will require changes
to the comment text that I have provided.
Your use of the term 'Auxiliary' as in AuxiliaryLayer and so on, conflicts with the use of the term
'auxiliary' as in 'auxiliary_output', and this could lead to a lot of confusion as the same word is
used in two very different senses.   I think it would be better to change all instances of
'auxiliary' (as used in this new sense) to 'existing'.  Of course names in code would have to
change as well as in comments.
If some sentences no longer make sense, you may have to reword them: don't just blindly
search and replace.